assets/runtime/functions

```diff
@@ -241,8 +241,12 @@ gitlab_finalize_redis_parameters() {
 }
 
 gitlab_check_redis_connection() {
+  REDIS_FLAG=""
+  if [[ -n ${REDIS_PASSWORD} ]]; then
+    REDIS_FLAG="-a ${REDIS_PASSWORD}"
+  fi
   timeout=60
-  while ! redis-cli -h ${REDIS_HOST} -p ${REDIS_PORT} -n ${REDIS_DB_NUMBER} ping >/dev/null 2>&1
+  while ! redis-cli -h ${REDIS_HOST} ${REDIS_FLAG} -p ${REDIS_PORT} -n ${REDIS_DB_NUMBER} ping >/dev/null 2>&1
```

Comment:
I'm quite not sure. If we should use a separate function for checking an authenticated redis connection.